## Velocity command for a differential drive on a flat surface

# +X direction - forward wrt robot frame
float32 translational_velocity

# -ccw direction, left wrt robot frame
float32 rotational_velocity
